lib/node_modules/@stdlib/blas/ext/base/dsum/manifest.json

```diff
@@ -1,42 +1,47 @@
 {
-	"options": {},
-	"fields": [
-		{
-			"field": "src",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "include",
-			"resolve": true,
-			"relative": true
-		},
-		{
-			"field": "libraries",
-			"resolve": false,
-			"relative": false
-		},
-		{
-			"field": "libpath",
-			"resolve": true,
-			"relative": false
-		}
-	],
-	"confs": [
-		{
-			"src": [
-				"./src/dsum.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [
-				"-lm"
-			],
-			"libpath": [],
-			"dependencies": [
-				"@stdlib/blas/ext/base/dsumkbn"
-			]
-		}
-	]
+  "options": {},
```

Comment:
We also need to add additional configuration to this file. See, e.g., https://github.com/stdlib-js/stdlib/blob/cf29ed0fc4f7108b5be01daca89ef72e6ebfcc1f/lib/node_modules/%40stdlib/blas/ext/base/dfill/manifest.json.